cinema/views.py

```diff
@@ -0,0 +1,38 @@
+from rest_framework.decorators import api_view
+from rest_framework.generics import get_object_or_404
+from rest_framework.response import Response
+from rest_framework import status
+
+from cinema.models import Movie
+from cinema.serializers import MovieSerializer
+
+
+@api_view(["GET", "POST"])
+def movie_list(request):
+    if request.method == "GET":
+        buses = Movie.objects.all()
+        serializer = MovieSerializer(buses, many=True)
+        return Response(serializer.data, status=status.HTTP_200_OK)
+    else:
+        serializer = MovieSerializer(data=request.data)
+        if serializer.is_valid():
+            serializer.save()
```

Comment:
Use `serializer.is_valid(raise_exception=True)` to simplify code. When you use `raise_exception=True`, if the serializer is invalid, an exception `(serializers.ValidationError)` is raised. DRF catches this exception and automatically returns a 400 Bad Request response with the appropriate error messages.